File: pagoServicios/src/app/modelos/pago.ts
import { MetodoDePago } from "./metodo-de-pago";

export interface Pago {
    id:number;
    descripcion:string;
    deuda:number;
    metodoDePagoId:number;
    metodoDePago:MetodoDePago;
    pagado:Boolean;
}

File: PagoServiceWebApi/Models/Pago.cs
using System;

namespace Models
{
    public class Pago : BaseEntity
    {
        public string Descripcion { get; set; }
        public decimal Deuda { get; set; }
        public bool Pagado { get; set; }
        public int MetodoDePagoId { get; set; }

        //Propiedad de Navegacion
        public virtual MetodoDePago MetodoDePago { get; set; }

    }
}
